app.py

```diff
@@ -222,10 +224,40 @@ def change_password():
 
 
 
-@app.get("/comment/<int:post_id>")
+@app.route("/comment/<int:post_id>", methods=['GET', 'POST'])
```

Comment:
Might this be better routed to `/posts/<int:post_id>/comments`? Usually, in REST, an identifier following a URL component indicates it belongs to that component (implying the user is querying a singular comment, not comments for a post).